modules/op-installing-pipelines-as-code-cli.adoc

```diff
@@ -0,0 +1,71 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
+[id="installing-pipelines-as-code-cli_{context}"]
+= Installing Pipelines as Code CLI 
+
+[role="_abstract"]
+In addition to installing Pipelines as Code on a {product-title} cluster, cluster administrators can install the `tkn-pac` CLI tool on local machines or as containers for testing.
+
+Currently, you can install the `tkn-pac` CLI tool using one or more of the following methods:
+
+* Install the latest stable release on any system where the `oc` or `kubectl` command line tool is available:
++
+[source,terminal]
+----
+$ VERSION=0.5.4
+$ oc apply -f https://raw.githubusercontent.com/openshift-pipelines/pipelines-as-code/release-$VERSION/release-$VERSION.yaml
+----
++
+[NOTE]
+====
+For the latest stable version, check the link:https://github.com/openshift-pipelines/pipelines-as-code/releases[release page]. In addition, check the {pipelines-title} release notes to ensure that the Pipelines as Code version is compatible with the {pipelines-shortname} version.
+====
++
+For latest but unstable features, you can install the development version:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/openshift-pipelines/pipelines-as-code/nightly/release.yaml
+----
+
+* Install on Linux or Mac OS using the `brew` package manager:
++
+[source,terminal]
+----
+$ brew install openshift-pipelines/pipelines-as-code/tektoncd-pac
+----
++
+You can upgrade the package by running the following command:
++
+[source,terminal]
+----
+$ brew upgrade openshift-pipelines/pipelines-as-code/tektoncd-pac
+----
+
+* Install as a container using `podman`:
++
+[source,terminal]
+----
+$ podman run -e KUBECONFIG=/tmp/kube/config -v ${HOME}/.kube:/tmp/kube \
+     -it quay.io/openshift-pipeline/pipelines-as-code tkn-pac help
+----
++
+Although not recommended, you can also use `docker` as a substitute for `podman`.
```

Comment:
If it's not recommended (or supported), then why mention this alternative at all? Consider omitting this sentence.